src/main/java/net/dv8tion/jda/api/entities/MessageSticker.java

```diff
@@ -0,0 +1,206 @@
+     */
+    @Nonnull
+    public String getPackId()
+    {
+        return Long.toUnsignedString(getPackIdLong());
+    }
+
+    /**
+     * The ID of the pack the sticker is from.
+     *
+     * @return the ID of the pack the sticker is from
+     */
+    public long getPackIdLong()
+    {
+        return packId;
+    }
+
+    /**
+     * The Discord hash-id of the sticker. This represents the actual asset file in the CDN for the sticker.
+     * <br><b>The URL for fetching sticker assets is currently private.
+     *
+     * @return the Discord hash-id of the sticker
+     */
+    @Nonnull
+    public String getAssetHash()
+    {
+        return asset;
+    }
+
+    /**
+     * The Discord hash-id of the preview of the sticker. This represents the actual preview asset file in the CDN for the sticker.
+     * <br><b>The URL for fetching sticker assets is currently private.
+     *
+     * @return the Discord hash-id of the preview image of the sticker or {@code null} if the sticker has no preview image
+     */
+    @Nullable
+    public String getPreviewAssetHash()
+    {
+        return previewAsset;
+    }
+
+    /**
+     * The url of the sticker asset.
+     *
+     * @return the url of the sticker
+     */
+    @Nonnull
+    public String getAssetUrl()
+    {
+        return String.format(ASSET_URL, id, asset, formatType.getExtension());
+    }
+
+    /**
+     * The {@link StickerFormat Format} of the sticker.
+     *
+     * @return the format of the sticker
+     */
+    @Nonnull
+    public StickerFormat getFormatType()
+    {
+        return formatType;
+    }
+
+    /**
+     * Set of tags of the sticker. Tags can be used instead of the name of the sticker as aliases.
+     *
+     * @return Possibly-empty unmodifiable Set of tags of the sticker
+     */
+    @Nonnull
+    public Set<String> getTags()
+    {
+        return tags;
+    }
+
+    public enum StickerFormat
+    {
+        /**
+         * The PNG format.
+         */
+        PNG(1, "png"),
+        /**
+         * The APNG format.
+         */
+        APNG(2, "apng"),
+        /**
+         * The LOTTIE format.
+         * <br>Lottie isn't a standard renderable image. It is a JSON with data that can be rendered using the lottie library.
+         *
+         * @see <a href="https://airbnb.io/lottie/">Lottie website</a>
+         */
+        LOTTIE(3, "json"),
+        /**
+         * Represents any unknown or unsupported {@link net.dv8tion.jda.api.entities.MessageSticker MessageSticker} format types.
+         */
+        UNKNOWN(-1, null);
+
+        private final int id;
+        private final String extension;
+
+        StickerFormat(final int id, final String extension)
+        {
+            this.id = id;
+            this.extension = extension;
+        }
+
+        public String getExtension()
+        {
+            return extension;
+        }
+
+        @Nonnull
+        public static MessageSticker.StickerFormat fromId(int id)
```

Comment:
```suggestion
        /**
         * Resolves the specified format identifier to the StickerFormat enum constant.
         *
         * @return The representative StickerFormat or UNKNOWN if it can't be resolved
         */
        @Nonnull
        public static MessageSticker.StickerFormat fromId(int id)
```